### PR TITLE
OLS-826: Add content describing controlling user access

### PR DESCRIPTION
**Pull request by rh-tokeefe**
Version(s): CP to TP1 branch

Issue:
https://issues.redhat.com/browse/OLS-826

Link to docs preview:
https://79144--ocpdocs-pr.netlify.app/openshift-lightspeed/latest/configure/ols-configuring-openshift-lightspeed.html#ols-about-lightspeed-and-role-based-access-control_ols-configuring-openshift-lightspeed

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by JoaoFula** [comment]
LGTM

File: modules/ols-granting-access-to-individual-users.adoc [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+.Procedure
+
+. Run the following command at the command line:
++
+[source,terminal]
+----
+oc adm policy add-cluster-role-to-user \
+lightspeed-operator-query-access <user_name>
+----
```
Comment (thoraxe) [inline modules/ols-granting-access-to-individual-users.adoc:26]: at some point in the future we should add the YAML for this, too.

**Comment by thoraxe** [comment]
LGTM

**Comment by rh-tokeefe** [comment]
Created OLS-866 to capture the action to add the file later on.

File: modules/ols-granting-access-to-individual-users.adoc (hunk 1 above)
Comment (rh-tokeefe) [reply modules/ols-granting-access-to-individual-users.adoc:26]: Created [OLS-866](https://issues.redhat.com//browse/OLS-866) to capture the action to add the file later on.

File: modules/ols-granting-access-to-individual-users.adoc [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+* You have access to the {ocp-short-name} CLI (oc).
+
+.Procedure
+
+. Run the following command at the command line:
++
+[source,terminal]
+----
+oc adm policy add-cluster-role-to-user \
```
Comment (gabriel-rh) [inline modules/ols-granting-access-to-individual-users.adoc:24]: I think is is more common to include the $ at the start of the command

File: modules/ols-granting-access-to-user-group.adoc [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+* You have access to the {ocp-short-name} CLI (oc).
+
+.Procedure
+
+. Run the following command at the command line:
++
+[source,terminal]
+----
+oc adm policy add-cluster-role-to-group \
```
Comment (gabriel-rh) [inline modules/ols-granting-access-to-user-group.adoc:24]: I think is is more common to include the $ at the start of the command

**Review by gabriel-rh — COMMENTED** [review]
lgtm - just minor comment

**Review by gabriel-rh — APPROVED** [review]
LGTM

**Pull request merged**